python/destinations/Prometheus-sink/library.json

```diff
@@ -0,0 +1,41 @@
+{
+	"libraryItemId": "8ba520a1-40f5-42e9-ab92-1f71ac717416",
+	"name": "Prometheus Sink",
+	"language": "Python",
+	"tags": {
+		"Complexity": [
+			"Medium"
+		],
+		"SDK Usage": [
+			"Read SDK"
+		],
+		"Pipeline Stage": [
+			"Destination"
+		],
+		"Technology": [
+			"Prometheus"
+		]
+	},
+	"shortDescription": "Prometheus metrics for a topic",
+	"DefaultFile": "main.py",
+	"EntryPoint": "build/dockerfile",
+	"RunEntryPoint": "main.py",
+	"Variables": [
+		{
+			"Name": "input",
+			"Type": "EnvironmentVariable",
+			"InputType": "InputTopic",
+			"Description": "Name of the input topic to listen to.",
+			"Required": true
+		}
+	],
+	"DeploySettings":
+	{
+		"DeploymentType": "Service",
+		"CpuMillicores": 200,
+		"MemoryInMb": 200,
+		"Replicas": 1,
+		"PublicAccess": false,
```

Comment:
Public access should be true here I suppose? Not sure how the name is picked for it however, so that is something to test.

Reply:
"DeploymentUrlPrefix": "your-name-here"